app/assets/stylesheets/landing.scss

```diff
@@ -0,0 +1,172 @@
+nav {
+  border-radius: 3%;
+  height: 5em;
+}
+
+.medium {
+  height: 45em;
+  width: 100%;
+  background: url("http://i.imgur.com/CShEgXJ.jpg") no-repeat;
+  background-size: cover;
+  display: block;
+}
+
+.row-break {
+  height: 4em;
+  width: 100%;
+}
+
+.cat-heading::before,
+.cat-heading::after {
+  position: absolute;
+  top: 50%;
+  width: 40%;
+  content: " ";
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  border-bottom: 1px solid #b2beb5;
+  display: block;
+  height: 0;
+}
+
+.card-desc > span {
+  font-size: 0.9em;
+  color: #000;
+}
+
+.card-desc > span.price {
+    color: #338574 !important;
+}
+
+.card,
+nav {
+  box-shadow: none !important;
+  -webkit-box-shadow: none !important;
+  -moz-box-shadow: none !important;
+}
+
+.card-image > img {
+  height: 25em;
+}
+
+.cat-heading:after {
+   right: 0;  
+   left: auto; 
+}
+
+.cat-heading:before {
+   right: auto;  
```

Comment:
Line should be indented 2 spaces, but was indented 3 spaces<br>Properties should be ordered left, right<br>Line contains trailing whitespace